### PR TITLE
Reserve one workaround for future use

### PR DESCRIPTION
**Pull request by jiaolu**

### CODE REVIEW

**Comment by amdvlk-admin** [comment]
### Test summary for commit 9d9d8d67f124b58f9268179b3e77fab64624325e
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 7387247eb9889ddcabbc1053b9c2052e253b088e</li>
<li>METROHASH: amd-master 712f76fee75d69b23a1ea8f6465752c3ccaaf9a2</li>
<li>PAL: dev b30561984a7b697ed7c709be76ff2e4d30532728</li>
<li>SPVGEN: dev b28e58f562591f335b9a3b5ae7e578a426df210b</li>
<li>XGL: dev 9bcc23fb29dab0b7ed1ebff17f3e99ae4da0bdf9</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev b3a0782762d195b57a2bebaabca24be90a35db2a</li>
</ul></details>

<details><summary>CTS tests (Failed: 0/430320)</summary><ul><li>Built with branch Vulkan-CTS-1.2.0.2-fetch-sources-fix</li>
<details open><summary>Rhel 8.2, Gfx8</summary><ul>

<li>Passed: 143394/143394 (100.0%)</li>
<li>Failed: 0/143394 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx10</summary><ul>

<li>Passed: 143438/143438 (100.0%)</li>
<li>Failed: 0/143438 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 143488/143488 (100.0%)</li>
<li>Failed: 0/143488 (0.0%)</li>
</ul></details>


</ul></details>

File: lgc/include/lgc/state/TargetInfo.h [hunk 1]
```diff
@@ -121,7 +121,8 @@ struct WorkaroundFlags {
       unsigned waWarFpAtomicDenormHazard : 1;
       unsigned waNggDisabled : 1;
       unsigned waLimitedMaxOutputVertexCount : 1;
-      unsigned reserved : 14;
+      unsigned : 1;
```
Comment (nhaehnle) [inline lgc/include/lgc/state/TargetInfo.h:124]: Please add a comment `// reserved for internal use`.

**Comment by amdvlk-admin** [comment]
### Test summary for commit aec0dc0f76e361d2be27f1f8be9cc6605b8e7f87
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 7387247eb9889ddcabbc1053b9c2052e253b088e</li>
<li>METROHASH: amd-master 712f76fee75d69b23a1ea8f6465752c3ccaaf9a2</li>
<li>PAL: dev b30561984a7b697ed7c709be76ff2e4d30532728</li>
<li>SPVGEN: dev b28e58f562591f335b9a3b5ae7e578a426df210b</li>
<li>XGL: dev 9bcc23fb29dab0b7ed1ebff17f3e99ae4da0bdf9</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev b3a0782762d195b57a2bebaabca24be90a35db2a</li>
</ul></details>

<details><summary>CTS tests (Failed: 1/426516)</summary><ul><li>Built with branch Vulkan-CTS-1.2.0.2-fetch-sources-fix</li>
<details open><summary>Rhel 8.2, Gfx8</summary><ul>

<li>Passed: 142126/142126 (100.0%)</li>
<li>Failed: 0/142126 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx10</summary><ul>

<li>Passed: 142170/142170 (100.0%)</li>
<li>Failed: 0/142170 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 142219/142220 (100.0%)</li>
<li><details><summary>Failed: 1/142220 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.pipeline.stencil.format.d16_unorm_s8_uint.states.fail_repl.pass_incc.dfail_decw.comp_equal
Stack trace: Flake
```
</p></details></li>


</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit 156e307aaa2982c15bedfcff62d09a862543904f
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 7387247eb9889ddcabbc1053b9c2052e253b088e</li>
<li>METROHASH: amd-master 712f76fee75d69b23a1ea8f6465752c3ccaaf9a2</li>
<li>PAL: dev b30561984a7b697ed7c709be76ff2e4d30532728</li>
<li>SPVGEN: dev b28e58f562591f335b9a3b5ae7e578a426df210b</li>
<li>XGL: dev 9bcc23fb29dab0b7ed1ebff17f3e99ae4da0bdf9</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev b3a0782762d195b57a2bebaabca24be90a35db2a</li>
</ul></details>

<details><summary>CTS tests (Failed: 0/426516)</summary><ul><li>Built with branch Vulkan-CTS-1.2.0.2-fetch-sources-fix</li>
<details open><summary>Rhel 8.2, Gfx8</summary><ul>

<li>Passed: 142126/142126 (100.0%)</li>
<li>Failed: 0/142126 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx10</summary><ul>

<li>Passed: 142170/142170 (100.0%)</li>
<li>Failed: 0/142170 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 142220/142220 (100.0%)</li>
<li>Failed: 0/142220 (0.0%)</li>
</ul></details>


</ul></details>

**Pull request merged**